### ISSUE TITLE
JSON format

### ISSUE THREAD

**Issue by kamicane** [estree/estree#82]
TLDR: are you interested in providing an official json format of the specification? I have made one, it lives here: https://github.com/kamicane/nodes/blob/master/spec.json

I recently started working again on my ast walk / validate / select library, [nodes](https://github.com/kamicane/nodes). I needed to update the es6 syntax since I was using a really old version from the mozilla website. I used to generate it with functions, but after a (short) [discussion](https://github.com/kamicane/nodes/issues/1) with @benjamn, i decided to try and make the spec into an easily consumable json format, that I now use to generate the (fake) multiple inheritance at runtime.

It would be fantastic if I could require the spec.json file directly from estree.

The only real problem I found with the specification (regarding using it to automatically generate classes from ast objects) lies with AssignmentProperty. Since it inherits from Property, but has "Property" as a type, it is impossible to know, from the ast, which one is it.

**Comment by RReverser** [comment]
There is already one that is generated from this spec, please see #60 and https://github.com/estree/formal/tree/master/formal-data.
